docs/filters.md

```diff
@@ -244,7 +244,7 @@ Matches issues of the given repository. If omitted, will match issues of the def
 
 ### sort
 
-*Expects a comma-separated list of sorting keys. For example, `repo, ~updated, -comments`. Any negation operator can be used to invert them. Keys can be any of `comments`, `repo`, `updated`, `date` (an alias for `updated`), `id`, `assignee` or a label group. Label groups can be disambiguated by appending a `.`.*
+*Expects a comma-separated list of sorting keys. For example, `repo, ~updated, -comments`. Any negation operator can be used to invert them. Keys can be any of `comments`, `repo`, `updated`, `date` (an alias for `updated`), `id`, `assignee`, `status`(`open` or `closed`) or a label group. Label groups can be disambiguated by appending a `.`.*
 
```

Comment:
`sort:priority` will arrange issues within the group by name (high, low, etc.) in alphabetical order, and put issues without that label group at the bottom.

If we're going to describe each input to `sort`, we should change the formatting and make it a list.